client/src/components/Form/Form.jsx

```diff
@@ -95,8 +127,152 @@ const Form = () => {
+          <MenuItem value="ME">Maine</MenuItem>
+          <MenuItem value="MH">Marshall Islands</MenuItem>
+          <MenuItem value="MD">Maryland</MenuItem>
+          <MenuItem value="MA">Massachusetts</MenuItem>
+          <MenuItem value="MI">Michigan</MenuItem>
+          <MenuItem value="MN">Minnesota</MenuItem>
+          <MenuItem value="MS">Mississippi</MenuItem>
+          <MenuItem value="MO">Missouri</MenuItem>
+          <MenuItem value="MT">Montana</MenuItem>
+          <MenuItem value="NE">Nebraska</MenuItem>
+          <MenuItem value="NV">Nevada</MenuItem>
+          <MenuItem value="NH">New Hampshire</MenuItem>
+          <MenuItem value="NJ">New Jersey</MenuItem>
+          <MenuItem value="NM">New Mexico</MenuItem>
+          <MenuItem value="NY">New York</MenuItem>
+          <MenuItem value="NC">North Carolina</MenuItem>
+          <MenuItem value="ND">North Dakota</MenuItem>
+          <MenuItem value="MP">Northern Mariana Islands</MenuItem>
+          <MenuItem value="OH">Ohio</MenuItem>
+          <MenuItem value="OK">Oklahoma</MenuItem>
+          <MenuItem value="OR">Oregon</MenuItem>
+          <MenuItem value="PW">Palau</MenuItem>
+          <MenuItem value="PA">Pennsylvania</MenuItem>
+          <MenuItem value="PR">Puerto Rico</MenuItem>
+          <MenuItem value="RI">Rhode Island</MenuItem>
+          <MenuItem value="SC">South Carolina</MenuItem>
+          <MenuItem value="SD">South Dakota</MenuItem>
+          <MenuItem value="TN">Tennessee</MenuItem>
+          <MenuItem value="TX">Texas</MenuItem>
+          <MenuItem value="UT">Utah</MenuItem>
+          <MenuItem value="VT">Vermont</MenuItem>
+          <MenuItem value="VI">Virgin Islands</MenuItem>
+          <MenuItem value="VA">Virginia</MenuItem>
+          <MenuItem value="WA">Washington</MenuItem>
+          <MenuItem value="WV">West Virginia</MenuItem>
+          <MenuItem value="WI">Wisconsin</MenuItem>
+          <MenuItem value="WY">Wyoming</MenuItem>
+      </Select>
+      </InputLabel>
+      <br></br>
+      <TextField
+        fullWidth
+        variant="filled"
+        error={zipError}
+        value={zip}
+        label="Zip"
+        InputLabelProps={{shrink: true}}
+        InputProps={{disableUnderline: true}}
+        onChange={(e) => {
+          setZip(e.target.value);
+          if (e.target.value.length > 5) {
+            setZipError(true);
+          }
+        }}
+        />
+      <TextField
+        fullWidth
+        variant="filled"
+        label="Mother's Maiden Name"
+        InputLabelProps={{shrink: true}}
+        InputProps={{disableUnderline: true}}
+        onChange={handleMotherMaidenName}
+        />
+      <TextField
+        type="phone"
+        fullWidth
+        variant="filled"
+        error={phoneError}
+        value={phone}
+        label="Phone Number"
+        onChange={(e) => {
+          setPhoneNumber(e.target.value);
+          if (e.target.value.length > 10) {
+            setPhoneError(true);
```

Comment:
Do you want to make it so that these errors can turn off once the phone input is back under 10 characters. Right now once you are over the 10 character limit, that field remains red even once you remove the invalid input.

The same is true for `setZipError` above.